appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml

```diff
@@ -124,13 +124,12 @@ limitations under the License.
       <plugin>
         <groupId>com.google.appengine</groupId>
         <artifactId>appengine-maven-plugin</artifactId>
-        <version>1.9.77</version>
+        <version>2.2.0</version>
         <configuration>
-          <!-- change to your app Id and version -->
-<!--
-          <appId>MY-APP-ID</appId>
-          <version>1</version>
- -->
+          <!-- set w/ -DprojectId=myProjectId on command line -->
+          <projectId>myProjectId</projectId>
+          <!-- "GCLOUD_CONFIG" allows for an autogenerated GAE version -->
+          <version>GCLOUD_CONFIG</version>
```

Comment:
You have the choice of specifying your own versions, or letting Google autogenerate the version.